app-backend/build.sbt

```diff
@@ -280,19 +299,23 @@ lazy val batch = project
       "com.fasterxml.jackson.module" % "jackson-module-scala_2.11" % "2.9.2"
     )
   })
-  .settings(assemblyShadeRules in assembly := Seq(
-    ShadeRule.rename("shapeless.**" -> "com.azavea.shaded.shapeless.@1").inAll,
-    ShadeRule
-      .rename(
-        "com.amazonaws.services.s3.**" -> "com.azavea.shaded.amazonaws.services.s3.@1"
-      )
-      .inAll,
-    ShadeRule
-      .rename(
-        "com.amazonaws.**" -> "com.azavea.shaded.amazonaws.@1"
-      )
-      .inAll
-  ))
+  .settings(
+    assemblyShadeRules in assembly := Seq(
```

Comment:
I'm pretty sure we don't even need this shade rule anymore because nothing is spark

Reply:
:man_shrugging: